src/components/Table/Table.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import Contact from '../Contacts/Contact'
+import contactsData from "../../contacts.json"
+
+const restContacts = contactsData.map(x => x);
+
+for (let i = 0; i < 5; i++) {
+  restContacts.shift();
+}
+
+function Table() {
+  const contactsArray = contactsData.slice(0, 5);
+  const [contacts, setContacts] = useState(contactsArray);
+
+
+
+  function handleAddRandomContact() {
+    if (restContacts.length !== 0) {
```

Comment:
```suggestion
    const restContacts = contactsData.filter(contact => !contacts.includes(contact));
    if (restContacts.length !== 0) {
```
Basicamente lo que le estamos diciendo es que se quede con todos aquellos contactos que no esten en el estado, por eso negamos el includes, porque si esta incluido daria true y ese no le queremos, queremos lo que el includes devuelva false (al negarlo true)